source/fe/mapping_fe_field.cc

```diff
@@ -202,6 +202,20 @@ MappingFEField<dim,spacedim,DoFHandlerType,VectorType>::InternalData::fourth_der
 
 
 
+namespace
+{
+  template<int dim>
+  Quadrature<dim> get_vertex_quadrature()
```

Comment:
add a quick doxygen documentation

Reply:
This is unnecessary. Its in the source and an anonymous namespace - its never exposed to the user.

Reply:
I never said that it's for users 😃 

Documentation of internal functions are also good to have so that we don't need to guess what's the purpose. Otherwise we might as well get rid of all in-code comments, no users will ever read them.

Reply:
> Otherwise we might as well get rid of all in-code comments, no users will ever read them.

Sigh. This was not particularly constructive.

> Documentation of internal functions are also good to have so that we don't need to guess what's the purpose. 

So, you recommend an inline comment to explain the purpose of this function, rather than making generated documentation?

Reply:
>  This was not particularly constructive.

the point was that we do document internal functions/algorithms.

> you recommend an inline comment to explain the purpose of this function

Does not matter which format it is (as it's never rendered) but a short one sentence explaining the function can never harm.